src/main/java/com/nuclei/assignment/Application.java

```diff
@@ -0,0 +1,88 @@
+package com.nuclei.assignment;
+
+import com.nuclei.assignment.constants.DisplayConstants;
+import com.nuclei.assignment.constants.FileConstants;
+import com.nuclei.assignment.enums.Field;
+import com.nuclei.assignment.enums.Order;
+import com.nuclei.assignment.models.User;
+import com.nuclei.assignment.services.storage.FileStorage;
+import com.nuclei.assignment.services.storage.MemoryImpl;
+import com.nuclei.assignment.services.storage.Storage;
+import com.nuclei.assignment.services.userservice.UserService;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Application {
+  private Storage file;
+  private Storage memory;
+  private UserService userService;
+  private Scanner sc;
+
+  /**
+   * initialize file and memory storing.
+   */
+  public Application() throws Exception {
+    this.file = new FileStorage(FileConstants.FILE_NAME);
+    this.memory = new MemoryImpl();
+    List<User> users = file.readAll(Field.NAME, Order.ASC);
+
+    if (Objects.nonNull(users) && users.size() != 0) {
+      this.memory.save(users);
+    }
+
+    this.sc = new Scanner(System.in);
+    this.userService = new UserService(sc, memory, file);
+  }
+
+  /**
+   * run method is used to take input from user.
+   * switch it to appropriate service function.
+   */
+  public void run() {
```

Comment:
methods should be doing one thing, if the objective of this function is to take input then why is named run?  

_"As you've probably read elsewhere, a function should do one thing, and only one thing.
Functions should do something, or answer something, but not both.”_ 

excerpt from clean code by Robert Cecil Martin.

Mandatorily read the 2nd chapter meaningful names and 3rd chapter Functions.